cpp/log/etcd_consistent_store-inl.h

```diff
@@ -314,6 +386,73 @@ std::string EtcdConsistentStore<Logged>::GetFullPath(
   return root_ + key;
 }
 
+
+// static
+template <class Logged>
+template <class T>
+Update<T> EtcdConsistentStore<Logged>::TypedUpdateFromWatchUpdate(
```

Comment:
Heh, this one is a bit more subtle - it's really only a member so that it can get `friend` access to `EntryHandle::EntryHandle()/SetHandle()` etc.